src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <title>Shows site app</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo">
+            <a href="#" id="refresh">
+                <img id="imgLogo" src="." alt="movie logo">
+            </a>
+        </div>
+        <nav>
+            <a href="#" class="showCounter">
+                <p>Movies</p>
+                <p id="count"></p>
+            </a>
+            <a href="#">Gender</a>
+            <a href="#">Year</a>
+        </nav>
+    </header>
+    <div class="container" id="container"></div>
+    <div class="" id="modalContainer"></div>
+    <div id="overlay"></div>
```

Comment:
- [ ] Please kindly make sure that the modal is scrollable, because at the moment the scrolling stops if you have a lot of comments and you might not be able to add new comments.